lib/govuk/lint/version.rb

```diff
@@ -1,5 +1,5 @@
 module Govuk
   module Lint
-    VERSION = "4.1.0".freeze
+    VERSION = "4.1.1".freeze
```

Comment:
This is more like a minor change, than a patch for a bug, but we're not to strict about this anyway.

Reply:
Yup! now bumping with a minor release